validation/connector_plugin_manifest_latest.xsd

```diff
@@ -96,11 +98,12 @@
   </xs:simpleType>
   <xs:group name="VendorInformation-G">
     <xs:sequence>
-      <xs:element minOccurs="0" name="vendor-information">
+      <xs:element name="vendor-information">
```

Comment:
Let's hold off on this file's change.  I'd rather you merge your pull request into dev, then we can update the packaging branch with the latest changes from dev to pick this up

Reply:
Sure, that makes sense. Fixed.